client/client.go

```diff
@@ -456,6 +456,8 @@ const (
 
 	ErrorKindSystemRestart = "system-restart"
 	ErrorKindDaemonRestart = "daemon-restart"
+
+	ErrorKindNoAssertion = "assertion-not-found"
```

Comment:
I assume you meant `ErrorKindAssertionNotFound` and not `ErrKindAssertionNotFound` since I can't find anywhere else we do `ErrKind`, we always seem to do `ErrorKind`.

Fixed to `ErrorKindAssertionNotFound` anyways